tensorflow/python/data/kernel_tests/interleave_test.py

```diff
@@ -402,6 +403,23 @@ def _build_dataset():
 
     self.run_core_tests(_build_dataset, 20)
 
+class SampleFromDatasetsCheckpointTest(checkpoint_test_base.CheckpointTestBase,
```

Comment:
please put this to tensorflow/python/data/experimental/kernel_tests/directed_interleave_dataset_test.py

Reply:
addressed in the commit.